object_manager_network_container.go

```diff
@@ -76,6 +76,33 @@ func (objMgr *ObjectManager) UpdateNetworkContainer(
 	return nc, nil
 }
 
+func (objMgr *ObjectManager) AllocateNetworkContainer(
+	netview string,
+	cidr string,
+	isIPv6 bool,
+	prefixLen uint,
+	comment string,
+	eas EA) (networkContainer *NetworkContainer, err error) {
+
+	containerInfo := NewNetworkContainerNextAvailableInfo(netview, cidr, prefixLen, isIPv6)
+	container := NewNetworkContainerNextAvailable(containerInfo, isIPv6, comment, eas)
+
+	ref, err := objMgr.connector.CreateObject(container)
+	fmt.Println(ref)
```

Comment:
Please, remove this.

Reply:
I have removed this debug print statement.